portlets/src/main/frontend/src/apps/profileStats/components/ProfileStats.vue

```diff
@@ -49,27 +51,38 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     </v-container>
     <achievements-drawer ref="achievementsDrawer" :user-points="userPoints" />
     <connections-drawer :connections-drawer="connectionsDrawer" :connection-requests="connectionRequests" :is-current-user-profile="isCurrentUserProfile" :common-connections="commonConnections" @closeDrawer="closeConnectionsDrawer"></connections-drawer>
-    <space-drawer :space-drawer="spaceDrawer" :space-requests="spaceRequests" @closeDrawer="closeSpaceDrawer" />
+    <space-drawer
+      :space-drawer="spaceDrawer"
+      :space-requests="spaceRequests"
+      :check-current-user-profile="checkCurrentUserProfile"
+      :commons-space-default-size="commonsSpaceDefaultSize"
+      @closeDrawer="closeSpaceDrawer" />
   </v-app>
 </template>
 <script>
-    import {getCommonConnections} from "../profilStatsAPI";
-
+   import {getCommonsSpaces, getCommonConnections} from '../profilStatsAPI';
     export default {
         data: function() {
           return {
             currentComponent: null,
             isFlipped: false,
             connectionsDrawer: false,
+            checkCurrentUserProfile: false,
```

Comment:
Could you rename it to **isCurrentUserProfile**
```suggestion
            isCurrentUserProfile: false,
```

Reply:
done!